docs/components/Typography/Typography.stories.mdx

```diff
@@ -0,0 +1,19 @@
+import { Canvas, Meta } from "@storybook/addon-docs";
+import { Typography } from "@jobber/components/Typography";
+import { Banner } from "@jobber/components/Banner";
+
+# Typography
+
+<Meta title="Components/Typography/Docs" component={Typography} />
+
+Base component that defines typographic standards across Jobber.
+
+Typography is the component that powers our Heading and Text components “under
+the hood”. You should first look to use
+[Heading](/docs/atlantis-text-and-typography-heading--heading) and
```

Comment:
Final comment, I think update the links to go to `Heading` and the docs page for `Text`

Reply:
Good catch! I fixed it